README.rst

```diff
@@ -45,3 +45,24 @@ Running `mypy`
 .. code-block:: console
 
     $ tox -e mypy
+
+
+Generating the documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: console
+
+    $ tox -e docs
+
+Note that this requires the `Enchant C library <https://pyenchant.github.io/pyenchant/install.html>` to be installed.
+
+*Note for M1 users, the* ``Enchant`` *library doesn't work out of the box and the workaround in the official docs is
+brittle, I suggest installing the library from* ``homebrew`` *and setting the environment variable* ``PYENCHANT_LIBRARY_PATH``
+*like so:*
+
+.. code-block:: console
+
+    $ export PYENCHANT_LIBRARY_PATH=/opt/homebrew/lib/libenchant-2.dylib
+
+*Keep in mind that you'll need to pass the environment variable to the tox environment*.
```

Comment:
I'm not sure what you mean by that exactly. If it means running `PYENCHANT_LIBRARY_PATH=/opt/homebrew/lib/libenchant-2.dylib tox -e docs`, than we'd need to add it to `pass_env` in `tox.ini`.

Reply:
Yes, but only in the case you're using an `m1`, I am guessing this works out of the box for you.

Reply:
Added.